scripts/ci/build-installers.sh

```diff
@@ -44,6 +44,10 @@ if [ -z "$ARGV_OPERATING_SYSTEM" ] || [ -z "$ARGV_ARCHITECTURE" ]; then
   usage
 fi
 
+if [ "${GH_TOKEN:-false}" != "false" ]; then
```

Comment:
So if `$GH_TOKEN` is set, we _automatically_ set `$PUBLISH` to 1? Why do we want this behaviour, instead of setting both `$GH_TOKEN` and `$PUBLISH` separately? (I guess it's easy to imagine `$GH_TOKEN` being stored as a global env-var, and then `$PUBLISH` only being selectively set on the command-line where we invoke `build-installers.sh` ?)